src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -94,6 +94,9 @@ public void Close()
             CloseWebSocket();
         }
 
+        public EndPoint? LocalEndPoint => _socket.LocalEndPoint;
```

Comment:
`_socket` will be null on mobile platforms, won't it?
```suggestion
        public EndPoint? LocalEndPoint => _socket?.LocalEndPoint;
```

Reply:
updated.